tests/integration/components/o-s-s/url-input-test.ts

```diff
@@ -28,7 +28,7 @@ module('Integration | Component | o-s-s/url-input', function (hooks) {
                                       @validationRegex={{this.subdomainRegex}} />`);
       await fillIn('input', 'fakedomai');
       await typeIn('input', 'n');
-      assert.true(this.onChange.calledOnceWithExactly('fakedomain', true));
+      assert.true(this.onChange.lastCall.calledWith('fakedomain', true));
```

Comment:
If the test needs to be updated, it means the logic has changed somewhere - is this expected ?